boards/arm/adafruit_grand_central_m4_express/support/openocd.cfg

```diff
@@ -0,0 +1,19 @@
+source [find interface/jlink.cfg]
+
+transport select swd
+
+set CHIPNAME atsamd51p20a
+source [find target/atsame5x.cfg]
+
+adapter speed 500
```

Comment:
this seems slow, and it doesn't look like they are tweaking the default (2MHz) speed in the arduino core? I realize you may have taken this from other same51 boards, but is it really needed?

If it is indeed necessary, maybe add a comment similar to the one in itsybitsy_m4_express?

Reply:
It does indeed work with the default speed.